xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVTB.cpp

```diff
@@ -160,6 +183,11 @@ bool CRendererVTB::RenderHook(int index)
   return true;
 }
 
+void CRendererVTB::AddSupportedHwRenderFormats()
```

Comment:
those are not needed anymore. the old code had a static renderer and decoder needed to adapt to renderer. now it is the other way round. the correct type of renderer is chosen and instantiated on ERenderFormat set be decoder.
https://github.com/FernetMenta/xbmc/blob/master/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp#L653